src/CliOptions.cs

```diff
@@ -86,6 +86,9 @@ public static Mono.Options.OptionSet InitCommandLineOptions()
 
             { "to|trustowncert", $"the own certificate is put into the trusted certificate store automatically.\nDefault: {TrustMyself}", (string s) => TrustMyself = s != null },
 
+            { "msec|maxsessioncount=", $"maximum number of paralel sessions.\nDefault: {MaxSessionCount}", (ushort i) => MaxSessionCount = i },
+            { "msuc|maxsubscriptioncount=", $"maximum number of subscriptions.\nDefault: {MaxSubscriptionCount}", (ushort i) => MaxSubscriptionCount = i },

```

Comment:
Question: We use ushort for the param here, but the SDK expects an int. Does setting it to a negative number have a meaning that might be useful?